flink-connectors/flink-connector-hive/src/test/resources/endpoint/hive_catalog.q

```diff
@@ -102,15 +102,15 @@ show current database;
 # test hive table with parameterized types
 # ==========================================================================
 
-describe hive.additional_test_database.param_types_table;
+describe additional_test_database.param_types_table;
```

Comment:
should be `additional_test_database.param_types_table` since Hive doesn't support to specific catalog.
`hive.additional_test_database.param_types_table` will fail in HiveParser and then fall back to Flink's parser.